cypress/e2e/main-page.cy.js

```diff
@@ -26,36 +19,37 @@ describe('Main page', function () {
     cy.contains('PyPI - License')
   })
 
-  it('Shows badge from category', function () {
-    visitAndWait('/category/chat')
+  it('Shows badges from category', function () {
+    visitAndWait('/badges')
 
-    expectBadgeExample(
-      'Discourse status',
-      'http://localhost:8080/badge/discourse-online-brightgreen',
-      '/discourse/status?server=https%3A%2F%2Fmeta.discourse.org'
-    )
-  })
+    cy.contains('Build')
+    cy.contains('Chat').click()
 
-  it('Customizate badges', function () {
-    visitAndWait('/')
+    cy.contains('Discourse status')
+    cy.contains('Stack Exchange questions')
+  })
 
-    cy.get(SEARCH_INPUT).type('issues')
+  it('Shows expected code examples', function () {
+    visitAndWait('/badges/static-badge')
 
-    cy.contains('/github/issues/:user/:repo').click()
+    cy.contains('button', 'URL').should('have.class', 'api-code-tab')
+    cy.contains('button', 'Markdown').should('have.class', 'api-code-tab')
+    cy.contains('button', 'rSt').should('have.class', 'api-code-tab')
+    cy.contains('button', 'AsciiDoc').should('have.class', 'api-code-tab')
+    cy.contains('button', 'HTML').should('have.class', 'api-code-tab')
+  })
 
-    cy.get('input[name="user"]').type('badges')
-    cy.get('input[name="repo"]').type('shields')
-    cy.get('table input[name="color"]').type('orange')
+  it('Build a badge', function () {
+    visitAndWait('/badges/git-hub-issues')
```

Comment:
Suspect this has always been present within the docusaurus ecosystem, but first time I'm noticing the hyphen-ization on the paths. Bit unfortunate that it's splitting "GitHub" this way :thinking:, assume that's unavoidable barring widespread changes to something like all the service class names?

Reply:
Yes I agree this is annoying.

I did have a look to see if there is a way we can customise the slug.

The slugs for these pages are generated by docusaurus-plugin-openapi here https://github.com/cloud-annotations/docusaurus-openapi/blob/0beb11b248e34e2b88f171406fc4fb04a390f065/packages/docusaurus-plugin-openapi/src/openapi/openapi.ts#L105-L114

They're derived by slugging the endpoint title through [lodash.kebabCase](https://lodash.com/docs#kebabCase).

So there would be no way to display "GitHub Issues" on the page but have it slug to `github-issues`.

Although this is suboptimal, I think we just roll with it.